packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -138,7 +138,7 @@ export class FocusTrapZone extends React.Component<IFocusTrapZoneProps, {}> impl
   }
 
   public focus() {
-    const { focusPreviouslyFocusedInnerElement, firstFocusableSelector } = this.props;
+    const { focusPreviouslyFocusedInnerElement, firstFocusableSelector, firstFocusableElement } = this.props;
```

Comment:
cc @czearing

Reply:
There's a separate PR (#14804) to convert this to a function component, so this change needs to be brought over there. @czearing can help with this once this is merged. (cc @ecraig12345 )